packages/rich-text-from-markdown/README.md

```diff
@@ -1,6 +1,6 @@
 # rich-text-from-markdown
 
-A Typescript library that converts markdown to rich text.
+A library to convert markdown to Rich Text document.
```

Comment:
Nit: The phrasing here is just a little awkward. I would rephrase to something like

```md
A library to convert markdown to Contentful rich text document format.
```